internal/confmapprovider/discovery/testdata/config.d/properties.discovery.yaml

```diff
@@ -0,0 +1,2 @@
+splunk.discovery.receivers.smartagent/postgresql.config.params.unused_param: param_value
+splunk.discovery.extensions.docker_observer.config.timeout: 1s
```

Comment:
I'm not following, what's the purpose of this file and the `invalid-properties.d/properties.discovery.yaml` file?

Reply:
They are used in tests `TestConfig` and `TestInvalidConfigDirContents`.